packages/preview-middleware-client/types/sap.ui.core.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'sap/ui/core/util/reflection/JsControlTreeModifier' {
+    import ChangeSelector from 'sap/ui/fl/ChangeSelector';
+    import Control from 'sap/ui/core/Control';
+    import Component from 'sap.ui.core.UIComponent';
```

Comment:
there is no fl/ChangeSelector, we only have a typedef for sap.ui.fl.Selector. UIComponent needs slashes instead if dots

Reply:
Fixed with: 748bd9d46c3ad60478c0f076c447af273d055d31